### PR TITLE
[Mobile Session Replay] Add stopRecording API

### PR DESCRIPTION
**Pull request by mariedm**
### What does this PR do? What is the motivation?
When we previously updated the documentation on mobile Session Replay in #25194, we forgot to mention the `stopRecording` API.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ val sessionReplayConfig = SessionReplayConfiguration.Builder([sampleRate])
 
 ### Start the recording manually
```
Comment (rtrieu) [inline content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md:223]:
```suggestion
### Start or stop the recording manually
```

Comment (rtrieu) [reply content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md:223]: I think adding this in the header will be good for anyone searching for this info.

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor suggestion, but otherwise looks good. Thanks for the PR!

File: content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md (hunk 1 above)
Comment (mariedm) [reply content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md]: Agreed! Thanks for the quick review!

**Review by rtrieu — APPROVED** [review]
LGTM

**Pull request merged**